Prepare a 3.0.0 release (#59)
softdevteam/vob: CHANGES.md

# vob 3.0.0 (2021-09-20)

## Breaking changes

* `Vob::from_elem(value: bool, len: usize)` now takes the value first and the
  number of repetitions of that value second to mirror the most common way this

>   function is defined elsewhere.

Reviewer: I think it would be better if this was slightly more explicit that we mean `from_elem` as it appears in e.g. `Vec`.

Author: Weirdly, `Vec` does not define `from_elem` -- even though other related types in the Rust repo do!

Reviewer: oh derp. disregard this then.

> * `Vob::from_elem(value: bool, len: usize)` now takes the value first and the
>   number of repetitions of that value second to mirror the most common way this
>   function is defined elsewhere.

Reviewer: ```suggestion
* `Vob::from_elem(value: bool, len: usize)` now takes the value first and the
  number of repetitions of that value second to mirror the most common way
  `from_elem` is defined elsewhere, such as in `Vec`.
```
